common/djangoapps/student/models.py

```diff
@@ -250,7 +250,7 @@ class Meta:  # pylint: disable=missing-docstring
     country = CountryField(blank=True, null=True)
     goals = models.TextField(blank=True, null=True)
     allow_certificate = models.BooleanField(default=1)
-    bio = models.TextField(blank=True, null=True)
+    bio = models.CharField(blank=True, max_length=3000, db_index=False)
```

Comment:
i think it makes sense if you choose to make it NOT NULL (i.e. `null=False`, which is the default so you could simply omit it everywhere).  Presumably django will default the value of this field to an empty string in all cases, which might be preferable in the code.